### PR TITLE
Feature/fix ena path

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR fixes ENA path construction logic to deal with IDs over 7 digits.

### CODE REVIEW

File: atlas-fastq-provider-functions.sh [hunk 1]
```diff
@@ -12,10 +12,19 @@ get_library_path() {
     local prefix=
     if ! [[ $subDir =~ "ENC" ]] && [[ -z "$forceShortForm" ]] ; then
         local num=${library:3}
-        
+       
+        # ENA pattern is:
+        # 
+        # - 6-digit codes under e.g. SRR1234567
```
Comment (suhaibMo) [inline atlas-fastq-provider-functions.sh]: You mean SRR123456 ?

Comment (pinin4fjords) [reply atlas-fastq-provider-functions.sh]: Yep!

**Review by suhaibMo — APPROVED** [review]
LGTM

**Comment by pinin4fjords** [comment]
Thanks @suhaibMo !

**Pull request merged**